### ISSUE TITLE
List of mounted SD card is not visible and exception is visible in terminal when user clicks on OK in 'Mounted SD Card' dialog

### ISSUE THREAD

**Issue by mmarciniak90** [opendatakit/briefcase#244]
#### Software versions 
Briefcase v1.8.0, 
Java v1.8.0_91
Operating system - Ubuntu

#### Problem description
* List of mounted SD card is not visible
* Exception is visible in terminal when user clicks on OK in 'Mounted SD Card' dialog

#### Steps to reproduce the problem
1. User opens Briefcase
2. User selects 'Mounted Android SD Card' in Pull data from field
3. User sees dialog 'Mounted SD Card'
4. List of mounted SD card is not visible
5. User clicks on OK button
6. NullPointerException is visible in terminal
![bug](https://user-images.githubusercontent.com/3907554/32945880-6a166b06-cb95-11e7-9821-13920bfe129c.png)

#### Expected behavior
User is able to select SD Card

**Comment by mmarciniak90** [comment]
Problem is visible only if 'Transfer file (MTP)' option is selected on Android device.
When 'Mass storage mode (MSC)' option is selected, Briefcase works as expected.

**Comment by yanokwa** [comment]
Agreed that this can be closed because we don't have the SD option anymore.
